src/main/java/model/IssuedLottos.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import model.generator.LottosGenerator;
+
+public class IssuedLottos {
+    private List<Lotto> lottos;
+
+    public IssuedLottos(LottosGenerator generator) {
+        this.lottos = Collections.unmodifiableList(generator.createLottos());
+    }
+
+    public static IssuedLottos merge(IssuedLottos from, IssuedLottos to) {
+        List<Lotto> collect = Stream.of(from, to)
+                .flatMap(issuedLottos -> issuedLottos.getLottos().stream())
```

Comment:
getter를 열어준 이유는 비즈니스 로직 수행이 아닌 View에 데이터를 전달하는 과정에만 사용하려고 했던 게 아니였나요?
도메인 내부에서는 필요하다면 직접 상태에 접근하는 게 어떨까요?
```suggestion
                .flatMap(issuedLottos -> issuedLottos.lottos.stream())
```